src/context/delay.test.ts

```diff
@@ -8,4 +8,11 @@ describe('delay', () => {
       expect(endResponse).toHaveProperty('delay', 1200)
     })
   })
+
+  describe('if no delay provided ', () => {
+    it('should set random delay on the response', () => {
+      const endResponse = response(delay())
+      expect(endResponse.delay).toBeDefined()
```

Comment:
What do you think if we check that:

```js
expect(endResponse.delay). toBeGreaterThan(MIN_SERVER_RESPONSE_TIME)
expect(endResponse.delay).toBeLessThan(MAX_SERVER_RESPONSE_TIME)
```

That way we ensure that the default delay value lies in the pre-defined boundaries.

Reply:
done!